src/jit/gentree.h

```diff
@@ -2887,6 +2887,7 @@ struct GenTreeCall final : public GenTree
         InlineCandidateInfo * gtInlineCandidateInfo;
         void *  gtStubCallStubAddr;             // GTF_CALL_VIRT_STUB - these are never inlined                
         CORINFO_GENERIC_HANDLE compileTimeHelperArgumentHandle; // Used to track type handle argument of dynamic helpers
+        void * gtDirectCallAddress; // Used to pass direct call address between lower and codegen
```

Comment:
I would suggest to initialize this field to nullptr in gtNewCallNode() and gtNewIndCallNode().

Further assert in codegenxarch.cpp that gtDirrectCallAddress is not nullptr.